### PR TITLE
Get universal picker url parameters via own method

### PR DESCRIPTION
**Pull request by rabauss**
see: https://github.com/contao/contao/pull/579#issuecomment-570549619

### CODE REVIEW

**Review by Toflar — DISMISSED** [review]
LGTM 👍 I asked @aschempp for another review though :)

**Review by aschempp — DISMISSED** [review]
It would probably require a custom picker builder and/or a custom DC_Table, but there‘s no reason not to add the flexibility 👍🏻

File: core-bundle/src/Resources/contao/widgets/Picker.php [hunk 1]
```diff
@@ -289,4 +285,20 @@ protected function getRelatedTable(): string
 
 		return (string) $arrRelations[$this->strField]['table'];
 	}
+
+	/**
+	 * Return the extra parameters for picker url
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/widgets/Picker.php]:
```suggestion
	 * Return the extra parameters for the picker URL
```

**Review by leofeyer — CHANGES_REQUESTED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @rabauss.

**Pull request merged**